web/src/components/apps/AppSnapshotSettings.jsx

```diff
@@ -352,7 +352,7 @@ class AppSnapshotSettings extends Component {
               </div>
             </div>
 
-            {useIam &&
+            {!useIam &&
```

Comment:
This change is intentional? Seems odd to reverse this logic without changing anything else